src/scripts/install-chromedriver.sh

```diff
@@ -162,7 +162,8 @@ else
   fi
   echo "$CHROMEDRIVER_VERSION will be installed"
 
-  if [[ $CHROMEDRIVER_VERSION -lt "121" ]]; then
+  CHROMEDRIVER_MAJOR_VERSION=$(echo $CHROMEDRIVER_VERSION | cut -d '.' -f1)
+  if [[ $CHROMEDRIVER_MAJOR_VERSION -lt 121 ]]; then
```

Comment:
Could $CHROME_RELEASE be exactly the string for that purpose?

Reply:
Came here as I also received the arithmetic error.

`/bin/bash: line 165: [[: 122.0.6261.69: syntax error: invalid arithmetic operator (error token is ".0.6261.69")`

I was going to propose`if [[ $CHROME_RELEASE -lt 121 ]]; then` check @hogelog mentioned above to be consistent with other checks in this script.

Reply:
hi @techsemicolon @hogelog 
I'm closing this PR since we've merged a different solution #109 and released v1.4.8 with this fix.

The error you faced should be resolved now if you update your circleci/browser-tools orb to latest @ v1.4.8:
https://circleci.com/developer/orbs/orb/circleci/browser-tools?version=1.4.8